examples/if-statement/if_statement.md

```diff
@@ -0,0 +1,8 @@
+# If statements
+
+If  statement is used for conditional execution. Conditions following `if` must be boolean and if the 
+condition is true, then the corresponding statement block is executed.
+
```

Comment:
We should say about, `else if` and `else` clauses as well. We can't assume users know theses.